src/tools/generateNick.ts

```diff
@@ -0,0 +1,3 @@
+export default function generateNick(username: string, connectionNubmer: number) {
```

Comment:
Any reason this can't just be where it's used directly? Assuming it's only used once

Reply:
Done already